src/Routes.js

```diff
@@ -43,12 +41,6 @@ const PackageDetail = lazy(() =>
     )
 );
 
-const NoAccess = lazy(() =>
-    import(
-        /* webpackChunkName: "NoAccess" */ './PresentationalComponents/NoAccessPage/NoAccess'
```

Comment:
shouldn't we also remove `'./PresentationalComponents/NoAccessPage/NoAccess'`? I see that there are no other files that import this component.